javascript/modules/client/README.md

```diff
@@ -752,6 +752,47 @@ for await (const step of steps) {
   }
 }
 ```
+### Checking if user can vote in an ERC20 proposal
+
+```ts
+import {
+  ClientErc20,
+  Context,
+  ContextPlugin,
+  CanVoteStep,
+  ICanVoteParams,
+} from "@aragon/sdk-client";
+
+// Create a simple context
+const context: Context = new Context(params);
+// Create a plugin context from the simple context
+const contextPlugin: ContextPlugin = ContextPlugin.fromContext(context);
+// Create an address list client
+const client = new ClientErc20(contextPlugin);
+
+const params: ICanVoteParams = {
+  address: "0x1234567890123456789012345678901234567890",
+  proposalId: "0x1234567890123456789012345678901234567890_0x1",
+  pluginAddress: "0x1234567890123456789012345678901234567890",
+};
+
+const steps = client.methods.canVote(voteParams);
+for await (const step of steps) {
+  try {
+    switch (step.key) {
+      case CanVoteStep.CHECKING:
+        console.log(step.txHash);
+        break;
+      case CanVoteStep.DONE:
+        console.log(step.canVote);
+        break;
+    }
+  } catch (err) {
+    console.error(err);
+  }
+}
```

Comment:
Why do we need a transaction here? This should be a read call and return the result immediately